### DISCUSSION TITLE
Please Help! I want to export the Cue Mix Points!

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#5381] [Q&A] [answered]
Hi guys,
Please tell me there is a solution to export cue/mix points.
I want to create a second station, but keeping the cue values for each song. How to export all those mix points that i edited in Visual Cue Editor?
Or maybe there is a way to set all this in the mp3 files metadata? So when i upload a file, Azura to recognise the mix points!

Please help!

**Comment by BusterNeece** [comment, marked as answer]
@unudelaradio As of the latest Rolling Release version, the Bulk CSV media export tool also exports any cue point data that was provided via the visual waveform editor.

**Reply by unudelaradio** [reply]
Wow! That's awesome! This option will be exactly what i need. Can't wait for the next version!
